frontend/src/sub-components/RoundLights.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import styled from "@emotion/styled";
+
+const Dot = styled.div`
+  display: flex;
+  height: 6rem;
+  width: 6rem;
+  border-radius: 50%;
+  margin-right: 10%;
+`;
+
+class RoundLights extends React.Component {
+  constructor(props) {
+    super(props);
+    this.statusColors = [];
+    /**
```

Comment:
JSDoc (`/**` then newline) should only be used for function/class documentation, not comments in general.

Reply:
done.